curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657f04ed0035f47ed04d0f1f.md

```diff
@@ -17,11 +17,12 @@ You should add a call to the `main()` function.
 
 ```js
 ({
-    test: () => {
-        const transformedCode = code.replace(/\r/g, "");
-        assert.match(transformedCode, /\n +main\s*\(\s*\)/);
-    }
-})
+  test: () => {
+    const pythonCode = `_Node(_code.replace('\\\\r', '')).has_call("main()")`;
```

Comment:
Okay, now it works and I'm going to sync. JFYI there's no need to replace `\r`.

Reply:
@larymak, there is one unresolved conversation. After that, the PR can be merged.